README.md

```diff
@@ -8,7 +8,7 @@ HTTP/2 Apple Push Notification Service for Rust using Tokio and async sending.
 
 ## Requirements
 
-Needs a Tokio executor version 1.0 or later and Rust compiler version 1.56.0 or later.
+Needs a Tokio executor version 1.0 or later and Rust compiler version 1.60.0 or later.
```

Comment:
What is the reason for the version bump in this case?

Reply:
The "dep:" syntax in features is only supported starting with 1.60 (https://doc.rust-lang.org/cargo/reference/features.html)

Reply:
👍🏼 cool no worries then